auth_jwt_demo/tests/spa/js/app.esm.js

```diff
@@ -1,4 +1,5 @@
-import {Oidc} from "./oidc-client.js";
+/* eslint-disable */
```

Comment:
this doesn't work here, any ideas?
I tried also:
`/* eslint-disable-line */`
`/* eslint-disable-next-line */`